src/store/socket_init.js

```diff
@@ -1,31 +1,77 @@
+/* eslint no-param-reassign: ["error", { "props": false }] */
 import io from 'socket.io-client';
-import * as actions from './Chat/actions.js';
-// import actions that have anything to do with sockets.
+import {
```

Comment:
No intentional reason anyway, but I guess my thought pattern was only import what I need, not the extras, and because I moved constants, actions, and the reducers into a single file I was hesitant to include everything to fall under the `actions.` as it didn't make sense to me reading it out loud. I think this may be the first drawback in my eyes to the ducks pattern?